### PR TITLE
HSF Planning 2023

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
Summary of the WG plans for 2023 - comments, clarifications and suggestions very welcome!

### CODE REVIEW

**Comment by hegner** [comment]
@graeme-a-stewart - is it draft /work in progress in the sense that you want to have a sign-off this Thursday?

**Comment by graeme-a-stewart** [comment]
> @graeme-a-stewart - is it draft /work in progress in the sense that you want to have a sign-off this Thursday?

Yes, exactly - I didn't want this merged before people had a chance to make their comments

File: organization/planning/plan-of-work-2023.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+* New topics
+  * Open data initiatives: Open data for reco studies (open thread by former conveners)
+  * Biased triggering and missed physics:
+    * Experiment TDAQ design and trigger/reco biases
+    * Strategies to “leave no stone unturned”
+  * Sustainability: Impact, indicators, better practices
+  * Focus on small/medium-sized experiments, e.g. Common DAQ/software frameworks
+  * Lessons learned” from new/upgraded experiments after 1-2 years of data-taking
+
+* Have a kick-off meeting in April (26?)
```
Comment (casarosa) [inline organization/planning/plan-of-work-2023.md:209]: the kick-off meeting is confirmed on the 26th of April, [indico](https://indico.cern.ch/event/1273894/).

Comment (graeme-a-stewart) [reply organization/planning/plan-of-work-2023.md]: Thanks! Updated

**Comment by klieret** [comment]
(Link checker is running into the limitations of the current workaround for #1046)

File: _includes/navbar.ext [hunk 2]
```diff
@@ -76,7 +76,7 @@
               <ul class="dropdown-menu" aria-labelledby="about_menu">
                 <li><a href="{{ site.baseurl }}/get_involved.html">Get involved!</a><li>
                 <li><a href="{{ site.baseurl }}/organization/team.html">The Coordination Team</a></li>
-                <li><a href="{{ site.baseurl }}/organization/planning/plan-of-work-2022.html">HSF Planning</a></li>
+                <li><a href="{{ site.baseurl }}/organization/planning/plan-of-work-2023.html">HSF Planning</a></li>
```
Comment (eduardo-rodrigues) [inline _includes/navbar.ext:79]: Does the 2022 plan file gets deleted or listed somewhere?

File: organization/planning/plan-of-work-2023.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+* Online workshop, we converged on the week of October 9‒13
+  * To be advertised soon, but see next slide …
+* Targeting as always the broader audience of developers and physicists:
+  * Idea of connecting Python users in HEP
+
+#### PyHEP Dev Workshop
+
+* New this year: [In person workshop for Python package developers](https://indico.cern.ch/event/1234156) @ Princeton, USA
+  * 25-29 July, at Princeton, aiming for ~40 people
+* *Idea to get developers getting their software products to work together as a coherent ecosystem*
```
Comment (eduardo-rodrigues) [inline organization/planning/plan-of-work-2023.md:95]:
```suggestion
* *Idea to get developers getting their software products to work together as a coherent ecosystem, and plan a coherent roadmap and make priorities for the upcoming year*
```

Comment (davidlange6) [reply organization/planning/plan-of-work-2023.md:95]: maybe consider a word other than coherent?

Comment (eduardo-rodrigues) [reply organization/planning/plan-of-work-2023.md:95]: This is what we wrote on the site. But feel free to suggest another word. Cc @jpivarski.

Maybe "interoperable"?

File: organization/planning/plan-of-work-2023.md [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+  * Needs careful design & large representative set of responses
+  * No recent comprehensive results available -> good to get overview of field
+* Set up a “Systematics challenge”
+  * Small analysis task to probe handling of systematic uncertainties
+  * Needs careful design (scope, effort involved to contribute)
+  * Would naturally frame further discussions on this topic following AEWII
+
+### Reconstruction and Software Triggers
+
+* Hot topics?
```
Comment (casarosa) [inline organization/planning/plan-of-work-2023.md:198]: we would like to remove lines 198 and 199

File: organization/planning/plan-of-work-2023.md [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+  * 2021 survey showed 4D reco and heterogeneous computing - still relevant today
+* New topics
```
Comment (casarosa) [inline organization/planning/plan-of-work-2023.md:200]: New topics -> 2023 Topics

File: organization/planning/plan-of-work-2023.md [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+  * Open data initiatives: Open data for reco studies (open thread by former conveners)
+  * Biased triggering and missed physics:
+    * Experiment TDAQ design and trigger/reco biases
+    * Strategies to “leave no stone unturned”
+  * Sustainability: Impact, indicators, better practices
+  * Focus on small/medium-sized experiments, e.g. Common DAQ/software frameworks
```
Comment (casarosa) [inline organization/planning/plan-of-work-2023.md:206]: remove this line please

File: organization/planning/plan-of-work-2023.md (hunk 5 above)
Comment (graeme-a-stewart) [reply organization/planning/plan-of-work-2023.md]: Should be ok now - please check

File: organization/planning/plan-of-work-2023.md [hunk 7]
```diff
@@ -0,0 +1,215 @@
...
+  * Small analysis task to probe handling of systematic uncertainties
+  * Needs careful design (scope, effort involved to contribute)
+  * Would naturally frame further discussions on this topic following AEWII
+
+### Reconstruction and Software Triggers
+
+* Hot topics?
+  * 2021 survey showed 4D reco and heterogeneous computing - still relevant today
+* New topics
+  * Open data initiatives: Open data for reco studies (open thread by former conveners)
```
[the remark this answers is not in the document]
Comment (graeme-a-stewart) [reply organization/planning/plan-of-work-2023.md]: Should be ok now - please check

File: _includes/navbar.ext (hunk 2 above)
Comment (eduardo-rodrigues) [reply _includes/navbar.ext:79]: Neat indeed! Thanks, I had not spotted those links from a quick read.

**Pull request merged**